pocs/camera/fli.py

```diff
@@ -306,15 +311,27 @@ def _fits_header(self, seconds, dark):
         return header
 
     def _get_camera_info(self):
-        self._info = {}
-        self._info['serial number'] = self._FLIDriver.FLIGetSerialString(self._handle)
-        self._info['camera model'] = self._FLIDriver.FLIGetModel(self._handle)
-        self._info['hardware version'] = self._FLIDriver.FLIGetHWRevision(self._handle)
-        self._info['firmware version'] = self._FLIDriver.FLIGetFWRevision(self._handle)
-        self._info['pixel width'], self._info['pixel height'] = self._FLIDriver.FLIGetPixelSize(self._handle)
-        self._info['array corners'] = self._FLIDriver.FLIGetArrayArea(self._handle)
-        self._info['array height'] = self._info['array corners'][1][1] - self._info['array corners'][0][1]
-        self._info['array width'] = self._info['array corners'][1][0] - self._info['array corners'][0][0]
-        self._info['visible corners'] = self._FLIDriver.FLIGetVisibleArea(self._handle)
-        self._info['visible height'] = self._info['visible corners'][1][1] - self._info['visible corners'][0][1]
-        self._info['visible width'] = self._info['visible corners'][1][0] - self._info['visible corners'][0][0]
+
```

Comment:
If you wanted to make this more succinct, you could omit the first 4 local vars, and could use `dict(serial_number=...` instead of `{'serial number': ...`